java/client/src/main/java/glide/connectors/handlers/ReadHandler.java

```diff
@@ -29,8 +30,7 @@ public void channelRead(@NonNull ChannelHandlerContext ctx, @NonNull Object msg)
     /** Handles uncaught exceptions from {@link #channelRead(ChannelHandlerContext, Object)}. */
     @Override
     public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
-        // TODO: log thru logger
-        System.out.printf("=== exceptionCaught %s %s %n", ctx, cause);
+        Logger.log(Logger.Level.ERROR, "read handler", "=== exceptionCaught " + ctx + " " + cause);
```

Comment:
you could import Logger.Level.ERROR
```suggestion
        Logger.log(ERROR, "read handler", "=== exceptionCaught " + ctx + " " + cause);
```